### PR TITLE
Fix own variable

### PR DESCRIPTION
**Pull request by TApplencourt**

### CODE REVIEW

File: bin/metababel [hunk 1]
```diff
@@ -274,20 +274,16 @@ base_folder = options[:folder] || "#{options[:component_type]}.#{options[:plugin
 folder = "#{base_folder}/metababel"
 FileUtils.mkdir_p(folder)
 
-d = { options: options }
+d = { options: options, params_declaration: nil, params_definition: nil }
```
Comment (DonAurelio) [inline bin/metababel:277]: I think this could be redundant, but I am not sure why did you add this

```ruby 
params_declaration: nil, params_definition: nil
```

Comment (TApplencourt) [reply bin/metababel:277]: We need it, or we need a default dict.

Comment (DonAurelio) [reply bin/metababel:277]: Ok, I understand, it is because the erb render requires the variable to be defined.

Comment (TApplencourt) [reply bin/metababel:277]: Exactly. It's not defaulted to `nil` as usual.

File: bin/metababel [hunk 2]
```diff
@@ -144,7 +142,12 @@ def wrote_event_dispatchers(folder, trace_model, trace_matching_model)
     # Modify the dispatcher to add the new dispatchType
     name_sanitized = sanitize(em.callback_name)
     dispatch_type_args = []
-    dispatch_type_args += [GeneratedArg.new('int64_t', '_timestamp')] if default_clock_class and default_clock_class.fetch(:extract, true)
```
Comment (DonAurelio) [inline bin/metababel:147]: I prefer the previous version (less lines), although I can live with the new one.

**Review by DonAurelio — COMMENTED** [review]
Seems good !!!

**Comment by TApplencourt** [comment]
Same! But robocopt changed it for me =( Maybe we can customize the config
file, but not that important. Like you I can live with you
